src/core/qgsfileutils.cpp

```diff
@@ -38,14 +38,15 @@ QString QgsFileUtils::representFileSize( qint64 bytes )
   list << QObject::tr( "KB" ) << QObject::tr( "MB" ) << QObject::tr( "GB" ) << QObject::tr( "TB" );
 
   QStringListIterator i( list );
-  QString unit = QObject::tr( "bytes" );
+  QString unit = QObject::tr( "B" );
```

Comment:
can you extend the unit tests to cover these changes?

Reply:
You're suggesting there should be unit tests to cover this function to begin with? ;)

Reply:
@nyalldawson , done.